client/javascript/views/addpatient.ejs

```diff
@@ -16,59 +16,84 @@
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+  
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item active px-2">
+          <a class="nav-link" href="/hos/registerpatient"><h3>Add a Patient or Doner</h3></a>
+        </li>
+        <li class="nav-item px-2">
+          <a class="nav-link" href="/hos/match"><h3>Find a match</h3></a>
+        </li>
+        
+        <li class="nav-item px-2">
+          <a class="nav-link" href="#"><h3>About</h3></a>
+        </li>
+      </form>
+    </div>
+  </nav>
+</header>
+<body class="container-fluid ">
+  <div style="min-height: 90%;"> 
+  <h1 style="padding-left: 5%;">Add a Patient or Donor</h1>
   <hr>
-  <form class="row g-3" action="/createpatient" method="post">
-
+  <form class="blockquote " action="/createpatient" method="post" style="padding-left: 5%;">
+<div class="row">
     <div class="col-md-2">
       <label class="form-label" for="autoSizingSelect">Type</label>
-      <select class="form-select" name="docType">
+      <select class="form-select" name="docType" required>
         <option value="patient">Patient</option>
         <option value="donor">Donor</option>
       </select>
     </div>
     <div class="col-md-2">
       <label for="name" class="form-label">ID</label>
-      <input type="text" class="form-control" name="PID">
+      <input type="text" class="form-control" name="PID" required>
     </div>
-
+    </div>
+    <div class="row">
     <div class="col-md-4">
       <label for="name" class="form-label">First Name</label>
-      <input type="text" class="form-control" name="firstName">
+      <input type="text" class="form-control" name="firstName" required>
     </div>
-     <div class="col-md-4">
-      <label for="name" class="form-label">First Name</label>
-      <input type="text" class="form-control" name="firstName">
+     <div class="col-md-3">
+      <label for="name" class="form-label">Middle Name</label>
+      <input type="text" class="form-control" name="middleName">
     </div>
     <div class="col-md-4">
       <label for="name" class="form-label">Last Name</label>
-      <input type="text" class="form-control" name="lastName">
+      <input type="text" class="form-control" name="lastName" required>
     </div>
-    <div class="col-md-2">
-      <label for="name" class="form-label">Age</label>
-      <input type="text" class="form-control" name="age">
     </div>
+    <div class="row">
     <div class="col-md-2">
       <label for="name" class="form-label">Phone</label>
-      <input type="text" class="form-control" name="phoneNumber" max="10">
+      <input type="text" class="form-control" name="phoneNumber" max="10" required>
```

Comment:
Make a provision that only numbers are allowed.

Reply:
done

Reply:
done